yt/frontends/swift/api.py

```diff
@@ -0,0 +1,26 @@
+"""
+API for SWIFT frontend
+
```

Comment:
Please reference the SWIFT code here and give a bit more information. Say that you worked on the file maybe so people know who to go and find when things don't work?